src/main/java/fi/aalto/cs/apluscourses/intellij/utils/CourseFileManager.java

```diff
@@ -145,6 +145,10 @@ private File getCourseFile(@NotNull Project project) {
         .toFile();
   }
 
+  public boolean courseFileExists(@NotNull Project project){
+    return getCourseFile(project).exists();
+  }
```

Comment:
Perhaps make this `synchronized`? Not 100% sure, but better safe than sorry :thinking: